moveit_ros/moveit_servo/package.xml

```diff
@@ -26,8 +26,9 @@
   <depend>control_msgs</depend>
   <depend>control_toolbox</depend>
   <depend>geometry_msgs</depend>
-  <depend>moveit_msgs</depend>
+  <depend>moveit_configs_utils</depend>
```

Comment:
This should be a test_depend

Reply:
i don't think so, these files are used when running the tutorials too

Reply:
Sorry, I meant for the moveit_kinematics not servo

Reply:
OK, that already is <test_depend>. Marking this as resolved